src/main/scala/io/appthreat/atom/package.scala

```diff
@@ -42,7 +43,8 @@ package object atom {
   case class AtomDataFlowConfig(
     sinkPatternFilter: Option[String] = None,
     mustEndAtExternalMethod: Boolean = false,
-    sliceDepth: Int = DEFAULT_SLICE_DEPTH
+    sliceDepth: Int = DEFAULT_SLICE_DEPTH,
+    unrollPaths: Boolean = false
```

Comment:
This is so cool! Shall we make it the default?

Reply:
Sure, that makes sense. Shall I remove the option entirely?

Reply:
Sure, we can remove the option.